conbench/tests/app/test_results.py

```diff
@@ -36,8 +39,46 @@ def test_legacy_route(self, client):
             r"unknown benchmark result ID: \w+", response.text, flags=re.ASCII
         )
 
+    @pytest.mark.parametrize(
+        "samples",
+        [(1,), (3, 5), (3, 5, 7)],
+    )
+    def test_display_result_no_mean(self, client, samples):
+        # Test that benchmark result view/page loads fine for the results that
+        # have one or two multisamples only, i.e. all aggregates are undefined.
+        self.authenticate(client)
+
+        result = _fixtures.VALID_RESULT_PAYLOAD.copy()
+        result["stats"] = {
+            "data": samples,
+            "times": [],  # key must be there as of now, validate more, change this.
+            "unit": "s",
+            "time_unit": "s",
+            # also see https://github.com/conbench/conbench/issues/813
+            # https://github.com/conbench/conbench/issues/533
+            "iterations": len(samples),
+        }
+
+        resp = client.post("/api/benchmark-results/", json=result)
+        assert resp.status_code == 201, resp.text
+        bmr_id = resp.json["id"]
+        resp = client.get(f"benchmark-results/{bmr_id}/")
```

Comment:
Interestingly, this did _not_ trigger the problem. So, needed to create the history scenario.